dali/operators/reader/loader/numpy_loader.cc

```diff
@@ -244,13 +73,13 @@ void NumpyLoader::ReadSample(NumpyFileWrapper& target) {
   auto current_file = FileStream::Open(path, read_ahead_, !copy_read_data_);
 
   // read the header
-  NumpyHeaderMeta header;
+  numpy::HeaderData header;
   auto ret = header_cache_.GetFromCache(filename, header);
   try {
     if (ret) {
       current_file->SeekRead(header.data_offset);
     } else {
```

Comment:
```suggestion
    if (!ret) {
```
and always seek, even after reading the header.